cpp/marc_tools/sed_command_convert_json_to_mrc.sed

```diff
@@ -0,0 +1,4 @@
+# sed command to use on json input data for convert_json_to_marc in order to unify data arrays
```

Comment:
json should be capitalized as JSON as it is an acronym standing for "JavaScript Object Notation".
Also, I would turn this into a bash script that converts the input file for convert_json_to_marc.  I think it is important to include CORE in the name and documentation for this file as we don't want to use this for any input for convert_json_to_marc but only for JSON files coming from CORE.